src/command_modules/azure-cli-maps/azure/cli/command_modules/maps/_params.py

```diff
@@ -38,6 +38,11 @@ def load_arguments(self, _):
         c.argument('tags',
                    arg_type=tags_type)
 
+    with self.argument_context('maps account create') as c:
+        c.argument('force',
```

Comment:
I'd recommend `--accept-tos` as it is clearer without even reading the help text what this will do.